tutorials/smart-contracts/deploy-erc20.md

```diff
@@ -0,0 +1,137 @@
+- A funded account with some WND tokens (you can get them from the [Westend Faucet](https://faucet.polkadot.io/westend?parachain=1000){target=\_blank})
+- Basic understanding of Solidity and fungible tokens
+
+## Creating the ERC20 Contract
+
+To create the ERC20 contract, you can follow the steps below:
+
+1. Navigate to the [Polkadot Remix IDE](https://remix.polkadot.io){target=\_blank}
+2. Click in the **Create new file** button under the **contracts** folder, and name your contract as `MyToken.sol`
+
+    ![](/images/tutorials/smart-contracts/deploy-erc20/deploy-erc20-1.webp)
+
+3. Now, paste the following ERC20 contract code into the editor
+
+    ```solidity title="MyToken.sol"
+    --8<-- 'code/tutorials/smart-contracts/deploy-erc20/MyToken.sol'
+    ```
+
+    The key components of the code above are:
+
+    - Contract Imports
+
+        - [`ERC20.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/token/ERC20/ERC20.sol){target=\_blank} - the base contract for fungible tokens, implementing core functionality like transfers, approvals, and balance tracking
+        - [`Ownable.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/access/Ownable.sol){target=\_blank} - provides basic authorization control, ensuring only the contract owner can mint new tokens
+    
+    - Constructor Parameters
+
+        - `initialOwner` - sets the address that will have administrative rights over the contract
+        - `"MyToken"` - the full name of your token
+        - `"MTK"` - the symbol representing your token in wallets and exchanges
+
+    - Key Function
+
+        - `mint(address to, uint256 amount)` - allows the contract owner to create new tokens for any address. The amount should include 18 decimals (e.g., 1 token = 1000000000000000000)
+        - Inherited [Standard ERC20](https://ethereum.org/en/developers/docs/standards/tokens/erc-20/){target=\_blank} functions:
+            - `transfer(address recipient, uint256 amount)` - sends a specified amount of tokens to another address
+            - `approve(address spender, uint256 amount)` - grants permission for another address to spend a specific number of tokens on behalf of the token owner
+            - `transferFrom(address sender, address recipient, uint256 amount)` - transfers tokens from one address to another, if previously approved
+            - `balanceOf(address account)` - returns the token balance of a specific address
+            - `allowance(address owner, address spender)` - checks how many tokens an address is allowed to spend on behalf of another address
+
+    !!! tip
+        Use the [OpenZeppelin Contracts Wizard](https://wizard.openzeppelin.com/){target=\_blank} to quickly generate customized smart contracts. Simply configure your contract, copy the generated code, and paste it into Polkadot Remix IDE for deployment.
+
+## Compiling the Contract
```

Comment:
Ah I just noticed this, but these titles should be action-oriented, so "Compile the Contract" instead.

- Use "Compiling the Contract" when describing the concept or providing general information
- Use "Compile the Contract" when giving a clear instruction in a tutorial or step-by-step guide

Can you please update the rest of the titles? I'll create a ticket to go back and check other guides